language/German/strings.po

```diff
@@ -2152,12 +2196,12 @@ msgstr "INIs"
 #: src/frontend/mame/ui/dirmenu.cpp:54
 msgctxt "path-option"
 msgid "UI Settings"
-msgstr ""
+msgstr "Benutzeroberfläche-Einstellungen"
```

Comment:
"Einstellungen der Benutzeroberfläche"? if not then as single word, without the hyphen: "Benutzeroberflächeneinstellungen"

Reply:
Fixed